filament/include/filament/IndirectLight.h

```diff
@@ -342,6 +343,68 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
     /** @deprecated use static versions instead */
     UTILS_DEPRECATED
     math::float4 getColorEstimate(math::float3 direction) const noexcept;
+
+    /**
+     * Sets IBL lookup options.
+     *
+     * @param options Options for IBL lookup settings.
+     */
+    void setOptions(IblOptions const& options) noexcept;
+
+    /**
+     * Gets IBL lookup options.
+     *
+     * @return IBL lookup options.
+     */
+    IblOptions const& getOptions() const noexcept;
```

Comment:
We don't need set/getOptions

Reply:
Fixed in b0f2ef543036f99a374befdcb44e38fbfa2b8df5 and 901f2a025aa0c1be79f2c56d51f7833d533bb4e4 + rebased it.